v1.9/stackpointio/PRODUCT.yaml

```diff
@@ -0,0 +1,6 @@
+vendor: StackPointCloud
+name: Stackpoint.io
+version: v1.0
+website_url: https://stackpoint.io
+documentation_url: https://stackpoint.io
+product_logo_url: https://github.com/StackPointCloud/spc-community/blob/master/app/assets/stackpointcloud.png
```

Comment:
We need an SVG. Is this one OK?

https://landscape.cncf.io/grouping=organization&organization=stack-point-cloud

Reply:
@dankohn yes, that one is fine. what steps do you need me to do?